### ISSUE TITLE
Lack of character varying(size) type

### ISSUE THREAD

**Issue by piotrbienias** [balderdashy/sails-postgresql#214]
Hello,

I am using the sails-postgresql in Hapi.js based project and I am a newbie in this topic. My consideration concerns creating a column having type of character varying with specified maximum size. In the attribute options I have defined the type 'string' and I thought that it would generate the character varying column, however the result was simply 'text' in the database. 

Is there any possibility to add character varying support?
I have browsed the source code and figured out something like that:

```javascript
// utils.js file (sqlTypeCast)
case 'string':
    return 'CHARACTER VARYING';
```

```javascript
// utils.js file (buildSchema):
var type = utils.sqlTypeCast(attribute.autoIncrement ? 'SERIAL' : attribute.type);
var size = attribute.size;

if (size !== undefined && type === 'CHARACTER VARYING') {
    type = type.concat('(' + size + ')');
}
```

**Comment by sailsbot** [comment]
Thanks for posting, @piotrbienias.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
